core-services/prow/02_config/_config.yaml

```diff
@@ -2465,6 +2462,54 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - operator-framework/operator-lib
+  - includedBranches:
+    - release-4.0
+    - release-4.1
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    labels:
+    - bugzilla/valid-bug
+    - cherry-pick-approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
+    repos:
+    - openshift/origin
+  - includedBranches:
+    - release-4.6
+    labels:
+    - group-lead-approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    repos:
+    - openshift/origin
+  - excludedBranches:
+    - release-3.11
+    - release-4.0
+    - release-4.1
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
```

Comment:
Also add bugzilla/invalid-bug. It means "if someone links a bug to this, it must not be invalid".

Reply:
got it,done